content/learn/maintained.adoc

```diff
@@ -0,0 +1,65 @@
+---
+menu:
+  learn:
+    parent: Workflow
+title: Validated Patterns - Maintained Tier
+weight: 44
+aliases: /requirements/maintained/
+aliases: /requirements/validated/
+---
+
+:toc:
+
+= The Validated Patterns 'Maintained' Tier 
+
+[id="tldr"]
+== tl;dr
```

Comment:
Maybe *Summary* ?

Reply:
could be. this is what the current pages used